debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/SourceInfoTest.java

```diff
@@ -18,8 +21,15 @@
 
     @Before
     public void beforeEach() {
```

Comment:
How about adding a test that asserts the entire source info as you've added it for the other connectors too?

Reply:
@gunnarmorling Added